src/mazela-artur/Fsd.Artur.Cs3/Fsd.Artur.Cs3/MainViev.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Fsd.Artur.Cs3.ShopActions;
+using Fsd.Artur.Cs3.WareHouse;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Artur.Cs3
+{
+    public class MainViev
+    {
+        public static void ShopViev(EnumAboutConsole isNew, Warehouse store)
+        {
+            if(isNew==EnumAboutConsole.Old)
+                Console.Clear();
+
+            Console.WriteLine($"What do you want?{Environment.NewLine}[1] Buy our product{Environment.NewLine}[2] Check our warehouse state{Environment.NewLine}[3] Add new product{Environment.NewLine}[4] Leave{Environment.NewLine}Decision:");
```

Comment:
Ja bym wolał to mieć w kilku liniach, aby było widać bez poziomego scrolla wszystkie wartości